### ISSUE TITLE
Seg fault when creating a new training input

### ISSUE THREAD

**Issue by kaboltzky** [semiautomaticgit/SemiAutomaticClassificationPlugin#15]
I encountered a QGis Segmentation fault when creating a new training input file.
At the moment I don't have a possibility to double check on another system but I
can reproduce this issue reliable on my current machine.

Let me know if you need more information.

(And **THANK YOU** for all your fantastic work with this plugin!)

**Steps to reproduce:**
* Open a new QGis-project
* Save the project
* Import some raster-datasets
* Create a BandSet
* Switch to the SCP-Dock, SCP-Input section
* Choose BandSet as Input
* Click Button "Create new training input"
* Enter a filename and confirm

**Exected behaviour:**
A new input-file should be created

**Actual behaviour:**
QGis crashes to Desktop. When launched from Terminal: `Segmentation fault (core dumped)`

**System and Versions:**

**OS**: Ubuntu 14.04 x64

**SCP**: 5.0.10

**Python**:
numpy: 1.11.1
scipy: 0.13.3
matplotlib: 1.5.1
gdal.VersionInfo(): 2010000

**QGIS**
QGIS version: 2.16.3
QGIS code revision: 7d00cae
Compiled against Qt: 4.8.6
Running against Qt: 4.8.6
Compiled against GDAL/OGR: 2.1.0
Running against GDAL/OGR: 2.1.0
Compiled against GEOS: 3.5.0-CAPI-1.9.0
Running against GEOS: 3.5.0-CAPI-1.9.0 r4084
PostgreSQL Client Version: 9.3.4
SpatiaLite Version: 4.1.1
QWT Version: 5.2.3
PROJ.4 Version: 480
QScintilla2 Version: 2.8.1

**Comment by semiautomaticgit** [comment]
Hello, and thank you for your detailed report.
About this issue I suspect it is related to QGIS compilation. 
You can try this version of the plugin and see at which point you get the crash
[SemiAutomaticClassificationPlugin.zip](https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/files/495593/SemiAutomaticClassificationPlugin.zip)

**Comment by kaboltzky** [comment]
Hi!
wow, that was a quick response ;)
I tested the above version and the crash happens after the MessageBox "9".
I also tried to debug the plugin but QGis crashes when reaching a breakpoint. Regardless in which file/module/line/... a breakpoint is set.

That makes me think that you are right and the problem is not related to SCP but to my QGis build.
So this issue can be closed.
I will investigate this further and reopen it if necessary.

Sry for the trouble and Thanks again ;)
